tests/all/manifest.rs

```diff
@@ -55,7 +55,7 @@ fn it_recognizes_a_map_during_depcheck() {
 fn it_creates_a_package_json_default_path() {
     let fixture = fixture::fixture(".");
     let step = wasm_pack::progressbar::Step::new(1);
-    wasm_pack::command::utils::create_pkg_dir(&fixture.path, &step).unwrap();
+    wasm_pack::command::utils::create_pkg_dir(&fixture.path, &PathBuf::from("pkg"), &step).unwrap();
```

Comment:
Nitpick: this can be `Path::new("pkg")`, which won't allocate like how creating a temporary `PathBuf` will.

Also, since we are always using the default relative out dir in these tests, it makes sense to add a helper test utility that always uses "pkg" to `tests/all/utils/manifest.rs` and use that here.